x-pack/plugin/logsdb/src/yamlRestTest/resources/rest-api-spec/test/30_logsdb_default_mapping.yml

```diff
@@ -280,3 +280,693 @@ create logsdb data stream with timestamp object mapping:
 
   - match: { error.type: "illegal_argument_exception" }
   - match: { error.reason: "composable template [logsdb-index-template] template after composition with component templates [logsdb-mappings] is invalid" }
+
```

Comment:
Maybe also introduce a test case where we only sort by `@timestamp`?

Reply:
Right...some integration like APM does it.